x/bank/client/cli/tx.go

```diff
@@ -60,3 +66,71 @@ func NewSendTxCmd() *cobra.Command {
 
 	return cmd
 }
+
+// NewMultiSendTxCmd returns a CLI command handler for creating a MsgMultiSend transaction.
+// For a better UX this command is limited to send one coin from one address to multiple addresses.
+func NewMultiSendTxCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use: "multi-send [from_key_or_address] [to_address_1, to_address_2, ...] [amount]",
+		Short: `Send funds from one account multiple others.
+		By default, sends the [amount] to each address of the list.
+		Using the '--split' flag, the [amount] is split equally between the addresses.
+		Note, the '--from' flag is ignored as it is implied from [from_key_or_address].
+		When using '--dry-run' a key name cannot be used, only a bech32 address.`,
+		Args: cobra.MinimumNArgs(4),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			cmd.Flags().Set(flags.FlagFrom, args[0])
+			clientCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+
+			coins, err := sdk.ParseCoinsNormalized(args[len(args)-1])
+			if err != nil {
+				return err
+			}
+
+			split, err := cmd.Flags().GetBool(FlagSplit)
+			if err != nil {
+				return err
+			}
+
+			totalAddr := types.NewInt(int64(len(args) - 2))
+			// coins to be received by the addresses
+			sendCoins := coins
+			if split && !coins.IsZero() {
```

Comment:
Why would coins be zero here? Shouldn't we have a check earlier to return if its zero?

Reply:
Good point!